k8s/fluentd-s3-scheduler.yml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd-s3-scheduler
+  namespace: classroom
+automountServiceAccountToken: true
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: fluentd-s3-scheduler
+  namespace: classroom
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: fluentd-s3-scheduler
+  namespace: classroom
+
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: classroom
+  name: fluentd-s3-scheduler
+spec:
+  schedule: "*/5 * * * *"
+  concurrencyPolicy: Replace
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: fluentd-s3-scheduler
+          automountServiceAccountToken: true
+          containers:
+          - name: scheduler
+            image: wildflowerschools/k8s-task-scheduler:v6
+            env:
+            - name: CONFIG_PATH
+              value: "/app/config/schedule.yaml"
+            - name: K8S_CLUSTER_NAME
+              value: "wework"
```

Comment:
@optimuspaul  -  I needed this env key to make the scheduler work. Could you help me understand how the Python scheduler container uses this cluster name value when I don't think the cluster name is actually named "wework"?

Reply:
Not sure that is what it is or if that var is used anywhere. I know that when I was testing it originally it was on a cluster called wework.

Reply:
oh I see, if `K8S_CLUSTER_NAME` is set then it loads in cluster config. doesn't matter what it is set to.